tests/unit/test_vedabase.py

```diff
@@ -26,8 +106,9 @@ def tearDown(self):
 
     def test_vedabase(self):
```

Comment:
Can we get support for testing the types/parent classes on vb and its, too? For instance, `vb.train` should return a particular kind of thing that is both a BaseVedaGroup and a subclass of a pytables object